abstract.txt

```diff
@@ -0,0 +1,23 @@
+Open data formats and software for gamma-ray astronomy
+======================================================
+
+Authors: tbd
+
+In gamma-ray astronomy, a lot of proprietary data formats and software exist
+that were developed for one specific mission or experiment. Especially in
+ground-based gamma-ray astronomy, imaging atmospheric Cherenkov telescope (IACT)
+data and software until now was private to the collaborations operating the
+telescopes. There is a general movement in science towards open data and
+software and specifically the next big IACT, the Cherenkov telescope array (CTA)
+will be operated as an open observatory.
+
+We have created a Github organisation at https://github.com/open-gamma-ray-astro
+where we are developing data format specifications. A public mailing list was
+set up at https://lists.nasa.gov/mailman/listinfo/open-gamma-ray-astro and a
+first face-to-face meeting on IACT high-level data model and formats took place
+in April 2016 in Meudon. We hope that this open multi-mission effort will help
+accelerate the development of open data formats and open-source software for
+gamma-ray astronomy, leading to synergies in the development of analysis codes
+and eventually better science results (reproducible, multi-mission). This poster
+will summarise what we have done so far and hopefully will solicit comments or
```

Comment:
I changed from "... hopefullly will ..." to "... has the goal to ...":
> This poster will summarise what we have done so far and has the goal to solicit comments and
future contributions from the gamma-ray astronomy community.

(I don't want to be presumptuous and claim "... will solicit comments ...")